outputs.tf

```diff
@@ -35,3 +35,8 @@ output "kp_public_endpoint" {
   description = "Instance public endpoint URL"
   value       = local.kp_endpoints["endpoints.public"]
 }
+
+output "cbr_rule_ids" {
+  description = "CBR rule ids created to restrict Key Protect"
+  value       = module.cbr_rule[*].rule_id
```

Comment:
please change to `value = one(module.cbr_rule[*].rule_id)` so output will be null if no rules present rather than the empty list the splat operator results in

Reply:
Can we use a condition based on the input variable instead of the result from the module?
`value = length(var.cbr_rules) > 0 ? module.cbr_rule[*].rule_id : null`